uiowa/uiowa
# Webform/Honeypot Updates

Webform [6.1.2 to 6.1.3](https://git.drupalcode.org/project/webform/-/compare/6.1.2...6.1.3?from_project_id=59750)

Honeypot [2.01 to 2.1.1](https://git.drupalcode.org/project/honeypot/-/compare/2.0.1...2.1.1?from_project_id=53761)

Fixes #4028

## To Test
- For the general module update, there were some configurations to repair, but just poke around normal webform config for issues. This stable release has been out since February.
- As a webmaster, I can add a file component to my webform limit the list of allowed file extensions without developer interaction, however if I try to add a non-allowed extension like exe, makerbot, billmurray, I won't be able to save the component and will get a message as to why. The component's help text provides the original list of allowed extensions should the editor wish to revert back.
- Limiting the list of allowed extensions, try the form as a user and try uploading a file that is not allowed. The error message should pop up and be limited to your limited list.
- To test honeypot, just confirm honeypot and antibot are being loaded for anonymous users on a webform. You can do this by inspecting the page looking for `honeypot` and `antibot` bits
